spec/integration/aws_rds_parameter_group_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'pry'
```

Comment:
let's not leave pry here please

Reply:
Sorry for that, It'll never happen again.